projects/plugins/jetpack/extensions/blocks/blog-stats/blog-stats.php

```diff
@@ -71,36 +71,39 @@ function load_assets( $attributes ) {
 		);
 
 		if ( isset( $data->views ) ) {
-			$views = $data->views;
+			$stats = $data->views;
 		}
 	} else {
 		$data = convert_stats_array_to_object(
 			( new WPCOM_Stats() )->get_stats( array( 'fields' => 'stats' ) )
 		);
 
-		if ( isset( $data->stats->views ) ) {
-			$views = $data->stats->views;
+		if ( $attributes['statsData'] === 'views' && isset( $data->stats->views ) ) {
+			$stats = $data->stats->views;
+		}
+
+		if ( $attributes['statsData'] === 'visitors' && isset( $data->stats->visitors ) ) {
+			$stats = $data->stats->visitors;
 		}
 	}
 
-	$label = $attributes['label'] ? $attributes['label'] : esc_html(
+	$fallback_label = $attributes['statsData'] === 'visitors' ? esc_html(
+		_n( 'visitor', 'visitors', $stats, 'jetpack' )
```

Comment:
```suggestion
		/* Translators: Number of visitors */
		_n( 'visitor', 'visitors', $stats, 'jetpack' )
```

Reply:
Let's try this 🤔

Reply:
Build is still failing 🤔

Reply:
Do you have any insights please @jeherve @anomiex? Thanks

Reply:
Looks a bit ugly, but I think it relates to where the comment goes: https://github.com/Automattic/jetpack/pull/28351#discussion_r1071083525

Just added a commit that will maybe fix it? Not sure.

Reply:
Was worth a try, but nope, issue seems to persist!

Reply:
You should be able to work around that by adding a dummy third argument, like it's done here:
https://github.com/Automattic/jetpack/blob/7693bbd1d11a9ad661dcf0c7043aa1bf65609741/projects/plugins/jetpack/extensions/blocks/map/styles.js#L22-L25

Reply:
Part of the confusion is that this isn't the code causing that error, it's the JS code in `edit.js`. I'll comment there with details.

You should be able to check locally with `jetpack build --production plugins/jetpack`, or in `projects/plugins/jetpack/` run `pnpm build-production-extensions`.